x/profile/client/rest/tx.go

```diff
@@ -0,0 +1,124 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/gorilla/mux"
+)
+
+func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/profiles", createProfileHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profiles/edit/{address}", editProfileHandler(cliCtx)).Methods("PUT")
```

Comment:
We can remove the `/edit` inside the path. The REST method is enough to distinguish between the calls.

```suggestion
	r.HandleFunc("/profiles/{address}", editProfileHandler(cliCtx)).Methods("PUT")
```